src/Invoice.php

```diff
@@ -80,6 +101,28 @@ public function setCustomer(Customer $customer)
         return $this;
     }
 
+    public function getCustomerName()
+    {
+        return $this->customerName;
+    }
+
+    public function setCustomerName($customerName)
+    {
+        $this->customerName = $customerName;
+        return $this;
+    }
+
+    public function getDebitCredit()
```

Comment:
Is this some kind of boolean or enum? Can you add types please?

Reply:
Fixed in #142